doc/Sklearn_docs/SKLEARN_KRR _vs_SHOGUN_KRR copy.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
I think this needs a different path.

I also wonder whether we really need to have a comparison inside the repository. I would either

* make a notebook that showcases Shogun's KRR (or better: add it to the existing KRR notebook with some explanations), inside the shogun repo.
* make a notebook that compares things as part of your application. For that you dont need to send a PR, but can rather just post the gist link in your application

Reply:
In the proposal you've mentioned about filling gaps in our notebooks by a PR. So I went ahead and did this. Anyway, I can add the contents inside our "regression.ipynb" that's what you suggest right?.

Reply:
yes, see below. Also before you do stuff, try to reflect for a second whether it is useful, and what it is for. That helps you focus your efforts.